### DISCUSSION TITLE
How to switch ds cores?

### DISCUSSION THREAD

**Discussion by RandomUser423423** [EmulatorJS/EmulatorJS#711] [Q&A]
Ever since you guys have changed the ds cores in the code editor from desume to melonds it has been very laggy for me. Is there any way to get back to the desume core. I'm not very good at coding, in fact, I barely even understand what ds cores are even, but it would be great to get it back to desume.

**Comment by ethanaobrien** [comment]
Open the html file in a text editor. Replace "nds" with "desmume"

**Reply by RandomUser423423** [reply]
It worked! Thank you so much!
